### PR TITLE
Simpler carousel indicators css

### PR DESCRIPTION
**Pull request by MartijnCuppens**
Pseudo elements are used to increase the hit area by 10px on top and bottom, but this can also be done with transparent borders which saves a bit css.

Demo: https://codepen.io/MartijnCuppens/pen/ajNXZj

### CODE REVIEW

**Comment by XhmikosR** [comment]
I definitely like this, but better wait for one more approval.

**Comment by XhmikosR** [comment]
Ping @andresgalante @mdo

File: scss/_carousel.scss
Comment (mdo) [inline scss/_carousel.scss]: Can we move to a variable for the `10px` value, and then use that here, on L196 and L187?

Comment (MartijnCuppens) [reply scss/_carousel.scss]: Changed it to a variable and used `box-sizing: content-box` instead of adding the hit area height to indicator height (this makes it possible to define the hit area height in another unit).

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**
